packages/connect-form/src/utils/validation.ts

```diff
@@ -21,10 +23,12 @@ export type FieldName =
 export type ConnectionFormError = {
   fieldName?: FieldName;
   fieldIndex?: number;
+  fieldTab?: ConnectFormTab;
   message: string;
 };
 
 export interface ConnectionFormWarning {
+  fieldTab?: ConnectFormTab;
```

Comment:
Just a super super minor thing: is a bit strange to have a `fieldTab` here, where there are no fields. As far as I can tell we also don't really have any warning that can be related to a specific tab yet.

Reply:
Removed warning indicators for now since we weren't using them. 👍